pages/common/pwsh.md

```diff
@@ -3,18 +3,26 @@
 > PowerShell Core is a cross-platform automation and configuration tool/framework.
 > More information: <https://docs.microsoft.com/powershell/>.
 
-- Start an instance of PowerShell:
+- Start an interactive shell session:
```

Comment:
Perhaps this will read better?
```suggestion
- Start an interactive PowerShell session:
```

Reply:
I prefer not to mention shell name because it's clear from tldr page title. ;)